### PR TITLE
Prepare Holocene Mainnet release

### PR DESCRIPTION
**Pull request by sebastianst**

### CODE REVIEW

**Review by geoknee — APPROVED** [review]
Approving from the superchain registry side of things. Looks like we need to cope with upstream op-geth changes before merging, so we should get another review on those changes when they are ready.

**Comment by protolambda** [comment]
We're reverting the op-geth withdrawals-root change, to derisk the Holocene roll out. Do not merge this yet, until the chain config changes have been dropped, and until the op-geth dependency has been updated to include the merge of https://github.com/ethereum-optimism/op-geth/pull/449

**Review by protolambda — APPROVED** [review]

**Pull request merged**
